scripts/make/update.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+script_path=$(dirname $0)
+working_dir=$(pwd)
+cd "$script_path"
+cd ../..
+repo_root=$(pwd)
+
+source "$repo_root/.build.env"
+
+if [ "$drupal_version" == "7" ]; then
```

Comment:
I remember spending quite a long time coming up with a pretty bulletproof order for doing these tasks which is this:
```	  drush @$(ENVIRONMENT) $(DRUSH_ARGS) cc drush && \
	  drush @$(ENVIRONMENT) $(DRUSH_ARGS) rr --no-cache-clear && \
	  drush @$(ENVIRONMENT) $(DRUSH_ARGS) cc drush && \
	  drush @$(ENVIRONMENT) $(DRUSH_ARGS) master-ensure-modules --no-cache-clear && \
	  drush @$(ENVIRONMENT) $(DRUSH_ARGS) updb && \
	  drush @$(ENVIRONMENT) $(DRUSH_ARGS) fra
```

It took me quite a while to come to that, but I'm absolutely certain it's the correct order of execution. Also, you'll need some way to pass in the environment so that these commands can be run from one of Acquia's post deploy scripts.

Reply:
guess there's no harm in a rr here :)

Reply:
I don't think whats being suggested is a million miles apart from what we have already so I've "blended" the two approaches.

Reply:
I'm pretty sure that there was a compelling reason to have updb run after master is exec'ed. I think it's possibly because it negates the need for you to have a seperate cc all operation at the end and makes the whole thing run a lot faster. 

Also, if what we're going for is standards here then I think we need to put the feature-revert all in. Every one of our Drupal 7 site must have it's features in a consistent state.

Reply:
On that same performance note I'd say that having master enabled is a given that we can assume and therefore avoid running it on every single deployment.